### PR TITLE
recipe for jack1

### PR DESCRIPTION
**Pull request by TURX**
This is for #4593 as a dependency.

### CODE REVIEW

**Comment by TURX** [comment]
There is still a lack of some functions: https://paste.ubuntu.com/p/y3NPGz3kwh/

**Comment by Begasus** [comment]
Check issue: https://github.com/haikuports/haikuports/issues/233

**Comment by TURX** [comment]
The build has gone successful, but a warning prevents this to be a package.

https://paste.ubuntu.com/p/Sp8Scf5VJ3/

```
Warning: 'NoneType' object has no attribute 'split'
```

File: media-sound/jack1/jack1-0.125.0.recipe [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	lib:libreadline$secondaryArchSuffix
+	lib:libsamplerate$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	jack1${secondaryArchSuffix}_devel
+	devel:libjack = 0.1.0 compat >= 0
+	devel:libjacknet = 0.1.0 compat >= 0
+	devel:libjackserver = 0.1.0 compat >= 0
```
Comment (Begasus) [inline media-sound/jack1/jack1-0.125.0.recipe]: Missing $secondaryArchSuffix

File: media-sound/jack1/jack1-0.125.0.recipe [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	"
+REQUIRES_devel="
+	jack1$secondaryArchSuffix == base
```
Comment (Begasus) [inline media-sound/jack1/jack1-0.125.0.recipe]: Missing $portVersion

Comment (TURX) [reply media-sound/jack1/jack1-0.125.0.recipe]: Where to add the $portVersion?

**Comment by TURX** [comment]
After building and installing jack1, Shotcut crashed.

Terminal log: https://paste.ubuntu.com/p/QHYMYvRpV4/
Error report: https://paste.ubuntu.com/p/gJvSSFrBrP/

How could I disable munlock()?

**Comment by korli** [comment]
Shotcut doesn't have a direct dependency on jack; mlt should then be built with jack. Maybe it's better first to disable the warning in shotcut, and investigate whether we can start with shotcut+mlt+sdl.

**Comment by Begasus** [comment]
mlt has been rebuild with extra modules that should take care of this, no need for jack ... (crash for shotcut doesn't mention jack)

**Comment by alaviss** [comment]
Why are we adding JACK to Haiku? Don't we already have Media Kit?

Also, this recipe doesn't seem to patch in a media backend for Haiku, so it wouldn't be able to do JACK things (pluggable outputs), making it rather useless.

**Comment by korli** [comment]
> Why are we adding JACK to Haiku? Don't we already have Media Kit?
> 
> Also, this recipe doesn't seem to patch in a media backend for Haiku, so it wouldn't be able to do JACK things (pluggable outputs, or even audio), making it rather useless.

One could use the network backend :)

**Comment by Begasus** [comment]
There is already a jack2 recipe (marked broken), closing this one too

**Pull request closed without merging**